pkgs/by-name/no/notonoto-hs/package.nix

```
@@ -0,0 +1,31 @@
+{
+  lib,
+  fetchzip,
+  stdenvNoCC,
+}:
+stdenvNoCC.mkDerivation rec {
+  pname = "notonoto-hs";
+  version = "0.0.3";
+
+  src = fetchzip {
+    url = "https://github.com/yuru7/NOTONOTO/releases/download/v${version}/NOTONOTO_HS_v${version}.zip";
+    hash = "sha256-TUlrASn7Lfyb9UoUk0ssV2j+ereqzRQAsnHCMkTL+04=";
+  };
+
```

```suggestion
  src = fetchFromGitHub {
    owner = "yuru7";
    repo = "NOTONOTO";
    tag = "v${version}";
    hash = "sha256-1dbx4yC8gL41OEAE/LNDyoDb4xhAwV5h8oRmdlPULUo=";
  };

  nativeBuildInputs = [ fontforge ];

  dontConfigure = true;

  preBuild = "patchShebangs generate.pe";
```

Nixpkgs builds from source whenever possible.

---

I am trying to build it from source but when using `ttautohint-py` it requires a C dll that comes with it but is still not detected when building the package, it is recognized when building the fonts locally.